dev/san_validator.sh

```diff
@@ -2,26 +2,58 @@
 
 cert_path="$1"
 
-# Check for empty certificate path
-if [ -z "$cert_path" ]; then
-    echo "Error: Certificate path not provided"
-    echo "usage:"
-    echo "bash san_validator.sh <certificate path>"
-    exit 1
-fi
+validate_cert() {
+    input_path="$1"
+    cert_info=$(openssl x509 -in "$input_path" -noout -text 2>/dev/null)
+    if echo "$cert_info" | grep -q "Subject Alternative Name"; then
+        return 0
+    else
+        return 1
+    fi
+}
+
+is_root_ca() {
+    input_path="$1"
+
+    #Check the certificate is RootCA is not
+    is_self_signed=$(openssl verify -CAfile "$cert" "$cert" 2>&1)
 
-# Check for the certificate file exists
-if [ ! -f "$cert_path" ]; then
-    echo "Error: Certificate file not found."
+    if [[ $is_self_signed == *error* ]]; then
```

Comment:
Add 
`shopt -s nocasematch` 
to make this regex case insensitive

Reply:
Done